presets/4.4/leds/quaddition TT leds.txt

```diff
@@ -0,0 +1,60 @@
+#$ TITLE: Default 4.3 LED settings
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: LEDS
+#$ STATUS: OFFICIAL
+#$ KEYWORDS: defaults, reset, leds, 533, spec, quaddiction, sugarK
+#$ AUTHOR: suagrK
+#$ DESCRIPTION: Resets LED settings to suit used VTX channels used by7 the quaddiction Tiny Trainer spec team 
+#$ DESCRIPTION: Options: LED, colour or mode_color resets.
+
+# led designations
+
+# led
+led 0
+led 0 2,3::COV:1
+led 1 3,3::COV:1
+led 2 4,3::COV:1
+led 3 5,3::COV:1
+led 4 6,3::COV:1
+led 5 7,3::COV:1
+led 6 8,3::COV:1
+led 7 7,5::COV:1
+led 8 8,6::COV:1
+led 9 9,7::COV:1
+led 10 10,8::COV:1
+led 11 11,9::COV:1
+led 12 12,10::COV:1
+led 13 13,11::COV:1
+led 14 14,12::COV:1
+
+
+# color
+color 0 0,0,0
+# color 1 red (R1) [<=5672]
+color 1 0,0,255
+# color 2 orange (R2) [>5672<=5711]
+color 2 30,0,255
+# color 3 green (R3, F1) [>5711<=5750]
+color 3 120,0,255
+#color 4 cyan (R4, F2, F3) [>5750<=5789]
+color 4 180,0,255
+# color 5 (not in use)
+color 5 0,0,0
+# color 6 blue (R5, F4, F5) [>5789<=5829]
+color 6 240,0,255
+# color 7 (not in use)
+color 7 0,0,255
```

Comment:
If (not in use) should it be
```suggestion
color 7 0,0,0
```
Same for below

Reply:
@sugaarK ?